core/src/test/scala/integration/kafka/api/BaseConsumerTest.scala

```diff
@@ -82,112 +86,19 @@ abstract class BaseConsumerTest extends IntegrationTestHarness with Logging {
   }
 
   @Test
-  def testAutoCommitOnRebalance() {
```

Comment:
Yes, that's right. Admittedly, I could have saved this for a separate patch. I can still do so if there are any objections.